backend/src/modules/Boards/service/Boards.service.spec.js

```diff
@@ -2,34 +2,34 @@ import { InMemoryBoardsRepository } from '../repository/inMemory/InMemoryBoards.
 import { Board } from './Board.js';
 import { BoardsService } from './Boards.service.js';
 
-describe('Boards service |', () => {
+describe('Board service | add new board', () => {
   const testBoard = new Board({
     boardName: 'Wroclaw',
     mapCoordinates: { latitude: 51.88569995139321, longitude: 17.02390643626451 },
     accessType: 'public',
     adminId: '507f191e810c19729de860ea',
     description: 'sample description',
     avatarUrl: 'https://firebasestorage.googleapis.com/sampleavatarurl',
+    announcements: [],
   });
   const testBoardWithoutContent = new Board({});
+  //Given
+  const inMemoryBoardsRepository = new InMemoryBoardsRepository();
+  const boardsService = new BoardsService(inMemoryBoardsRepository);
 
-  test('when correct board is being added, then body is returned', async () => {
-    //Given
-    const inMemoryBoardsRepository = new InMemoryBoardsRepository();
-    const boardsService = new BoardsService(inMemoryBoardsRepository);
+  afterEach(() => {
+    boardsService.removeAllBoards();
+  });
```

Comment:
Zauważ, że tworzenie tutaj repozytorium spowoduje, że będziesz operować na tym samym repo w każdym "test", a nie na osobnym :)
Czyli jak w jednym teście coś dodamy to w innym teście będzie to dostępne.
Raczej powinniśmy testy robić niezależnymi :)

I wtedy nie musimy używać tego w afterEach.
(jest jeszcze taka zasada, że bazę danych powinniśmy czyścić przed testem, a nie po.
W skrócie chodzi o to, że gdy mamy problem i nie wiemy, dlaczego test nam nie przechodzi to jesteśmy w stanie sprawdzić recordy w bazie danych — a gdy czyścimy ją po teście, to nie jesteśmy w stanie sprawdzić, czy operacje w DB przebiegły pomyślnie :)